### DISCUSSION TITLE
How to build Budibase source code from some pull requests

### DISCUSSION THREAD

**Discussion by shaikhashraf709** [Budibase/budibase#1171] [Help] [answered]
I am into a project where i am designing a large form for which i need some features in budibase which are not yet in main branch so i decided to build budibase from a pull request but i do not have so much experience in building an electronJS app that budibase is written in (if i am right). i searched on internet but its confusing. 

Thank you.

**Comment by mike12345567** [comment, marked as answer]
Hi @shaikhashraf709 - the form builder which I assume you're looking to use will be released tomorrow, 25th Feb - there will be a release available with all of our latest work available in it.

As for building from source, building an actual Electron app, in the `packages/server` directory you can run the command `yarn build:electron` which will build the platform correct version and output it to the `packages/server/dist` directory. However this requires you have setup the repo fully, using the information found [here](https://github.com/Budibase/budibase/blob/master/.github/CONTRIBUTING.md).

If you are simply hoping to use the form builder and all of the latest features I would recommend waiting for the release tomorrow, we wouldn't recommend using a build from a PR to build a real app as our Cloud will not be able to support it yet (when trying to deploy)  or you would need to build your self hosted configuration from source.

**Reply by shaikhashraf709** [reply]
Ok. i think i should wait for tomorrow.
